pramen/core/src/main/scala/za/co/absa/pramen/core/rdb/PramenDb.scala

```diff
@@ -63,6 +63,10 @@ class PramenDb(val jdbcConfig: JdbcConfig,
     if (dbVersion < 3) {
       initTable(MetadataRecords.records.schema)
     }
+
+    if (0 < dbVersion && dbVersion < 4) {
```

Comment:
Did you have something in mind with '0<..'? Otherwise,
```suggestion
    if (dbVersion < 4) {
```

Reply:
Don't add the same column twice. 
In case version is 0, the full table will be created in the initTable method
where `if (dbVersion < 1)`

Reply:
Cool, makes sense, thanks!